### PR TITLE
BIP-CTV: fix mistypes, omissions, etc

### PR DESCRIPTION
**Pull request by dgpv**
The fixes for issues spotted after first read

### CODE REVIEW

**Comment by dgpv** [comment]
Note that in "====NOP-Default and Standardness Rules====" I changed the `exact match for a 32-byte previously pushed value` to `If the argument length is not exactly 32`, because as I understand, it refers to the rule in Summary "The element on the stack is 32 bytes long, NOP otherwise", and the contents of the value must actually match the hash, if the length is correct.

File: bip-ctv.mediawiki [hunk 1]
```diff
@@ -358,7 +358,7 @@ designers.
 
 However, committing to the index eliminates key-reuse vulnerability to the half-spend problem. As
 potentially reused CHECKTEMPLATEVERIFY scripts commit to being spent at the same index, they cannot
-be spent at the same index. This makes it safer to design wallet vault contracts without half-spend
+be spent at the different index. This makes it safer to design wallet vault contracts without half-spend
```
Comment (JeremyRubin) [inline bip-ctv.mediawiki]: Actually same is correct here.

if you want to make this one more clear it should read "they cannot both be spent at the same index, which implies that they cannot be spent in the same transaction."

**Review by JeremyRubin — CHANGES_REQUESTED** [review]
Thanks!

I swear you can read something hundreds of times and miss the same little errors :)

**Comment by dgpv** [comment]
I rephrased the paragraph about the same index. please check.

File: bip-ctv.mediawiki [hunk 2]
```diff
@@ -356,10 +356,10 @@ Committing to the currently executing input's index is not strictly needed for a
...
 designers.
 
-However, committing to the index eliminates key-reuse vulnerability to the half-spend problem. As
-potentially reused CHECKTEMPLATEVERIFY scripts commit to being spent at the same index, they cannot
-be spent at the same index. This makes it safer to design wallet vault contracts without half-spend
-vulnerabilities.
+However, committing to the index eliminates key-reuse vulnerability to the half-spend problem.
+As CHECKTEMPLATEVERIFY scripts commit to being spent at particular index, reused instances of these
+scripts cannot be spent at the same index, which implies that they cannot be spent in the same transaction.
+This makes it safer to design wallet vault contracts without half-spend vulnerabilities.
```
Comment (dgpv) [inline bip-ctv.mediawiki:362]: Rephrased.

**Comment by JeremyRubin** [comment]
Thanks :)

**Pull request merged**